label.py

```diff
@@ -30,10 +32,21 @@ def get_new_image():
         return
     current_image = images_to_show.pop()
     print("data/%s_firefox.png" % current_image)
-    img = ImageTk.PhotoImage(Image.open("data/%s_firefox.png" % current_image))
+
+    img_firefox = Image.open("data/%s_firefox.png" % current_image)
+    img_chrome = Image.open("data/%s_chrome.png" % current_image)
+
+    background = img_firefox.convert("RGBA")
+    overlay = img_chrome.convert("RGBA")
+    img_blended = Image.blend(background, overlay, 0.5)
+
+    img = ImageTk.PhotoImage(img_blended)
```

Comment:
Sorry, one last nit, it would be good to move this after panel1 and panel2, just to have them sorted (I know the behavior doesn't change)